packages/cloud-cognitive/src/components/CreateModal/CreateModal.stories.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button,
+  TextInput,
+  NumberInput,
+  TextArea,
+  Dropdown,
+  DatePicker,
+  DatePickerInput,
+  RadioButton,
+  RadioButtonGroup,
+} from 'carbon-components-react';
+
+import { pkg } from '../../settings';
+import '../../utils/enable-all'; // must come before component is imported (directly or indirectly)
+import { getStorybookPrefix } from '../../../config';
+import { CreateModal } from '.';
+import mdx from './CreateModal.mdx';
+const storybookPrefix = getStorybookPrefix(pkg, CreateModal.displayName);
+
+import styles from './_storybook-styles.scss';
+
+export default {
+  title: `${storybookPrefix}/${CreateModal.displayName}`,
+  component: CreateModal,
+  parameters: {
+    styles,
+    docs: {
+      page: mdx,
+    },
+  },
+};
+
+const Template = ({ children, ...args }) => {
+  return (
+    <CreateModal {...args} open>
```

Comment:
I found a problem with this, if you leave the prop `open` here, in the storybook controls I can't change it. It will always stay open.

Reply:
I would just let {...args} do its thing, and in the default args for this story just set open:true.

Reply:
@AlexanderMelox that is correct -- it's supposed to stay `open` just as it is in the Carbon storybook for modal, found here: https://react.carbondesignsystem.com/?path=/story/components-composedmodal--default

Reply:
Okay! I just found it weird when I tried clicking cancel in the first two stories for create modal and nothing happened. Maybe we should have somewhere in the story name like Opened Create modal?

Reply:
@AlexanderMelox we went over this in scrum and all agree that this isn't the best way to show the stories -- I went ahead and changed it :D